students/12th/hokyunkim/login.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Westagram</title>
+  <link href="style/login.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+  <div class="login">
+    <div class="login_pic">
+      <img src="img/app_phone.png" class="phone">
+    </div>
+    <div class="login_main">
+      <div class="login_body">
+        <div class="login_header">
+          <img src="img/logo_text.png">
+        </div>
+        <div class="login_form">
+          <div class = "textid">
+            <input type="text" placeholder="전화번호, 사용자 이름 또는 이메일">
+          </div>
+          <div class ="pass">
```

Comment:
```suggestion
          <div class="textid">
            <input type="text" placeholder="전화번호, 사용자 이름 또는 이메일">
          </div>
          <div class="pass">
```
속성과 값 사이에 띄어쓰기 없도록 전부 바꿔주세요